Improve docs on CI infra correlation (#34013)
DataDog/documentation: content/en/continuous_integration/pipelines/gitlab.md

{{% /tab %}}

{{% tab "Docker Autoscaler" %}}

> CI Visibility supports Infrastructure metrics for "Docker Autoscaler" executors through log-based correlation. Make sure GitLab job logs are indexed so Datadog can link jobs to hosts, and ensure those logs include messages in the form `Instance <hostname> connected`. Logs for GitLab jobs include the `datadog.product:cipipeline` and `source:gitlab` tags, which you can use in [Log Indexes][2] filters. Users also need [log read access][3] to see the Infrastructure data in this scenario. For more information, see the [Correlate Infrastructure Metrics with GitLab Jobs guide][1].

Reviewer: ```suggestion
CI Visibility supports Infrastructure metrics for "Docker Autoscaler" executors through log-based correlation. To enable this, make sure GitLab job logs are indexed so Datadog can link jobs to hosts, and that the logs include messages in the form `Instance <hostname> connected`. GitLab job logs include the `datadog.product:cipipeline` and `source:gitlab` tags, which you can use in [Log Indexes][2] filters. Users also need [log read access][3] to see the Infrastructure data in this scenario. For more information, see the [Correlate Infrastructure Metrics with GitLab Jobs guide][1].
```
